app/Helpers/TerrafundDashboardQueryHelper.php

```diff
@@ -135,8 +135,8 @@ public static function getPolygonsByStatusOfProjects($request)
         $projectUuids = TerrafundDashboardQueryHelper::buildQueryFromRequest($request)
             ->pluck('v2_projects.uuid');
 
-        $requestedStatuses = $request->input('statuses');
+        $approvedStatus = ['approved'];
```

Comment:
why are you using an array here?

Reply:
Because the other function to retrieve data, receives an array of all statuses needed. 
Whay should I do in that case?